plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/internal/SQLEditorMessages.properties

```
@@ -194,8 +194,8 @@ pref_page_sql_format_label_format_active_query_tip = Formats only active query o
 pref_page_sql_completion_group_sql_assistant = SQL assistant/completion 
 pref_page_sql_completion_label_enable_auto_activation = Enable auto activation
 pref_page_sql_completion_label_enable_auto_activation_tip = Enables content assistant auto activation (on text typing)
-pref_page_sql_completion_label_enable_experimental_features = Enable experimental features
-pref_page_sql_completion_label_enable_experimental_features_tip = Enable beta version of new autocompletion enhancements for select statements
+pref_page_sql_completion_label_advanced_object_references_analyser = Use the advanced objects references analyser
```

We mostly use American English versions of words, so it should be "analyzer", not "analyser".

```suggestion
pref_page_sql_completion_label_advanced_object_references_analyser = Use the advanced objects references analyzer
```